packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -7,6 +7,10 @@ import { KeytipData } from '../../KeytipData';
 export interface ICheckboxState {
   /** Is true when Uncontrolled control is checked. */
   isChecked?: boolean;
+  isIndeterminate?: boolean;
+  /** Since indeterminant props are always truthy,
+   * isIndeterCheck state acts as a secondary logic gate to turn off indeterminate state after the first click */
+  isIndeterCheck?: boolean;
```

Comment:
Very confused why you would need this prop?

Reply:
It was to help the internal logic differentiate between the first click (to remove indeterminate state) versus successive clicks but I have since changed the logic to only work with this.state.isIndeterminate.